screens/AuthLoadingScreen.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+    ActivityIndicator,
+    AsyncStorage,
+    StatusBar,
+    StyleSheet,
+    View
+} from 'react-native';
+
+export default class AuthLoadingScreen extends React.Component {
+    constructor(props) {
+        super(props);
+        this._bootstrapAsync();
+    }
+
+    _bootstrapAsync = async () => {
+        const userToken = await AsyncStorage.getItem('userToken');
+        this.props.navigation.navigate(userToken ? 'Main' : 'Auth');
+    };
+
+    render() {
+        return (
+            <View>
```

Comment:
i took this from the react native docs, we'll have to style it later when we figure out how we want it to look.